contrib/buildsystems/CMakeLists.txt

```diff
@@ -178,10 +192,19 @@ endif()
 
 find_program(MSGFMT_EXE msgfmt)
 if(NOT MSGFMT_EXE)
-	set(MSGFMT_EXE ${CMAKE_SOURCE_DIR}/compat/vcbuild/vcpkg/downloads/tools/msys2/msys64/usr/bin/msgfmt.exe)
-	if(NOT EXISTS ${MSGFMT_EXE})
-		message(WARNING "Text Translations won't be built")
-		unset(MSGFMT_EXE)
+	if(NO_GETEXT)
+		if(MSGFMT_EXE)
```

Comment:
I fear that this is dead code, given that we're inside a top-level `if(NOT MSGFMT_EXE)` :grinning: